mlir/utils/jenkins/Jenkinsfile.miopen-test

```diff
@@ -33,55 +75,17 @@ pipeline {
                     }
                 }
                 stages {
-                    stage('Install parallel (temp until dockerfile updated)') {
-                        steps {
-                            sh 'apt-get update 2>&1 >/dev/null && apt-get install -y --no-install-recommends parallel 2>&1 >/dev/null'
-                        }
-                    }
                     stage('Environment') {
                         steps {
                             echo "Config is ($DTYPE, $LAYOUT, xdlops=$XDLOPS, dir=$DIRECTION)"
                             sh 'cat /etc/os-release'
                             sh '/opt/rocm/bin/rocm-smi'
                         }
                     }
-                    stage('Build libMLIRMIOpen') {
-                        steps {
-                            checkout scm
-                            cmakeBuild generator: 'Unix Makefiles',\
-                               buildDir: 'build',\
-                               sourceDir: 'llvm',\
-                               installation: 'InDocker',\
-                               buildType: 'Release',\
-                               cmakeArgs: '''-DLLVM_ENABLE_PROJECTS="mlir;lld"
-                    -DCMAKE_CXX_COMPILER=/opt/rocm/llvm/bin/clang++
-                    -DCMAKE_C_COMPILER=/opt/rocm/llvm/bin/clang
-                    -DBUILD_SHARED_LIBS=OFF
-                    -DLLVM_BUILD_LLVM_DYLIB=OFF
-                    -DLLVM_ENABLE_TERMINFO=OFF
-                    -DCMAKE_INSTALL_PREFIX=/opt/rocm
-                    --log-level=WARNING
-                    -DCMAKE_RULE_MESSAGES=OFF
-                    '''
-                            sh 'cd build; make -j $(nproc) libMLIRMIOpen all install >/dev/null'
-                        }
-                    }
-                    stage('Build MIOpenDriver') {
+                    stage("Copy MIOpen binaries") {
                         steps {
-                            dir('MIOpen') {
-                                git branch: 'develop', poll: false, url: 'https://github.com/ROCmSoftwarePlatform/MIOpen.git'
-                                cmakeBuild buildDir: 'build',\
-                                   buildType: 'Release',\
-                                   installation: 'InDocker',\
-                                   cmakeArgs: '''-DMIOPEN_USE_MLIR=On
--DCMAKE_CXX_COMPILER=/opt/rocm/llvm/bin/clang++
--DCMAKE_C_COMPILER=/opt/rocm/llvm/bin/clang
--DBUILD_DEV=On
--DMIOPEN_TEST_FLAGS=\'--verbose --disable-verification-cache\'
---log-level=WARNING
--DCMAKE_RULE_MESSAGES=OFF
-'''
-                                sh 'cd build; make -j $(nproc) MIOpenDriver'
+                            dir("MIOpen") {
+                                unstash name: "MIOpen-build"
```

Comment:
This copies the build step package to test stage? Nice!

Reply:
Yep.